tests/unit/test_storage.py

```diff
@@ -36,6 +36,34 @@ def test_get_service_info(self):
         storage_01ceph.get_service_info()
         self.assertEqual(storage_01ceph.CEPH_INFO, result)
 
+    @mock.patch.object(storage_01ceph, "CEPH_INFO", {})
+    def test_get_ceph_versions_mismatch(self):
+        result = {'versions': {
+                  'mgr': ['14.2.11'],
+                  'mon': ['14.2.11'],
+                  'osd': ['14.2.11'],
+                  'rgw': ['14.2.11']
+                  }}
+        storage_01ceph.get_ceph_versions_mismatch()
+        self.assertEqual(storage_01ceph.CEPH_INFO, result)
+
+    @mock.patch.object(storage_01ceph, "CEPH_INFO", {})
+    def test_get_ceph_pg_imbalance(self):
+        result = {'pgs-per-osd': {
+                   'osd.0': 295,
+                   'osd.3': 214,
+                   'osd.15': 49,
+                   'osd.16': 316,
+                   'osd.17': 370,
+                   'osd.34': 392,
+                   'osd.37': 406,
+                   'osd.56': 209,
+                   'osd.72': 206}}
+        storage_01ceph.get_ceph_pg_imbalance()
+        print(result)
+        print(storage_01ceph.CEPH_INFO)
```

Comment:
can you remove the print statements from these tests? (fyi i think you only see their output if they go to stderr anyway)